index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link  href="style.css" rel="stylesheet">
+   
+    <title>Zuhra Hashimi - Portfolio</title>
+</head>
+
+<body>
+    <div>
+           <nav class="topnav">
+                <a href="#home" class="logo">Zuhra Hashimi</a>
```

Comment:
- [ ] please try to stick to the Figma design by using the same color and font for your logo  and make sure the logo is center vertically 
![image](https://user-images.githubusercontent.com/61790587/171284303-6b822c79-11b2-4b87-a931-0ad98297df77.png)

Reply:
Thanks so much.
it's done.